src/flood_model/bg_flood_model.py

```diff
@@ -131,7 +131,7 @@ def run_model(
         bg_path, dem_path, catchment_boundary, resolution, end_time, output_timestep, rain_input_type
     )
     os.chdir(bg_path)
-    subprocess.call([bg_path / "BG_Flood_Cleanup.exe"])
+    subprocess.call([bg_path / "BG_flood.exe"])
```

Comment:
Is this a change from the new version or did you rename it? if you renamed it this could make it harder to update but it would make sense if cyprien had changed the name since this one makes more sense

Reply:
change from the new version.